ai-ml/emr-spark-rapids/install.sh

```diff
@@ -9,6 +9,7 @@ targets=(
   "module.vpc_endpoints"
   "module.eks"
   "module.eks_blueprints_addons"
+  "amp_ingest_irsa"
```

Comment:
remove this if its not needed or add `module. amp_ingest_irsa `

Reply:
Checked in all repos and fixed.